camayoc/qcs_models.py

```diff
@@ -84,7 +84,7 @@ def create(self, **kwargs):
         the data associated with this object's ``self._id``.
         """
         response = self.client.post(self.endpoint, self.payload(), **kwargs)
-        self._id = response.json()['id']
+        self._id = response.json().get('id')
```

Comment:
If the response is a 4xx for example the ID is not available so it will be set to None